common/src/main/java/apoc/result/IndexConstraintNodeInfo.java

```diff
@@ -34,18 +34,18 @@ public class IndexConstraintNodeInfo {
      * @param label
      * @param properties
      * @param status status of the index, if it's a constraint it will be empty
-     * @param type if it is an index type will be "INDEX" otherwise it will be the type of constraint
+     * @param schemaType if it is an index type will be "INDEX" otherwise it will be the type of constraint
```

Comment:
Actually not, because the output in handled by the field, so `public final String type;` in this case.
Now I have a doubt, is this what you meant by "internally"? 
If not, I can reset it :)

Reply:
Oh, you're correct! This is good then :)